src/playerClass.ts

```diff
@@ -12,11 +14,34 @@ export class Player {
     }
     this._name = name;
     this._pokemons = pokemons;
+    this._activePokemon = 0;
+    this._alivePokemons = pokemons;
```

Comment:
I think we don't need to duplicate the list `this._pokemons`, we can use filter on` this.pokemons` in the method `get alivePokemons()`